documentation/asciidoc/topics/proc_creating_caches_batch.adoc

```diff
@@ -0,0 +1,43 @@
+[id='creating_caches_batches-{context}']
+= Creating Caches in Batches
+Add multiple caches with batch operations with the {brandname} CLI.
+
+.Prerequisites
+
+* Download the server distribution so you can run the CLI.
+* Retrieve the necessary client connection details.
```

Comment:
Maybe we could list here what the connection details are?

Reply:
There's a section in the docs "11.2. Client Connection Details" which I think covers it instead of being too repetitious. Perhaps that's not the best approach though. Users might miss that. Let me know what you think when you review the rendered HTML.